web/engines/py_engine/services/profiles.py

```diff
@@ -1254,6 +1256,502 @@
+                        "age": 84,
+                        "has_dutch_nationality": True,
+                        "has_partner": False,
+                        "residence_address": "Zorgcentrum De Linde, Lindestraat 10, 1015RS Amsterdam",
+                        "has_fixed_address": True,
+                        "household_size": 1,
+                    }
+                ],
+                "relaties": [{"bsn": "999993805", "partnerschap_type": "GEEN", "partner_bsn": None}],
+                "verblijfplaats": [
+                    {
+                        "bsn": "999993805",
+                        "straat": "Lindestraat",
+                        "huisnummer": "10",
+                        "postcode": "1015RS",
+                        "woonplaats": "Amsterdam",
+                        "type": "WOONADRES",
+                    }
+                ],
+            },
+            "JenV": {
+                "wilsbekwaamheid": [
+                    {
+                        "bsn": "999993805",
+                        "status": "WILSONBEKWAAM",
+                        "sinds": "2022-10-28",
+                    }
+                ],
+                "vertegenwoordiging": [
+                    {
+                        "bsn": "999993805",
+                        "type": "MENTORSCHAP",
+                        "vertegenwoordigers": [
+                            {
+                                "bsn": "999993801",
+                                "naam": "Jan Vertegenwoordiger",
+                                "sinds": "2022-11-15",
+                            }
+                        ],
+                    }
+                ],
+            },
+            "BELASTINGDIENST": {
+                "box1": [
+                    {
+                        "bsn": "999993805",
+                        "loon_uit_dienstbetrekking": 0,
+                        "uitkeringen_en_pensioenen": 1950000,
+                        "winst_uit_onderneming": 0,
+                        "resultaat_overige_werkzaamheden": 0,
+                        "eigen_woning": 0,
+                    }
+                ],
+                "box2": [{"bsn": "999993805", "reguliere_voordelen": 0, "vervreemdingsvoordelen": 0}],
+                "box3": [
+                    {
+                        "bsn": "999993805",
+                        "spaargeld": 12500000,
+                        "beleggingen": 0,
+                        "onroerend_goed": 0,
+                        "schulden": 0,
+                    }
+                ],
+                "buitenlands_inkomen": [{"bsn": "999993805", "bedrag": 0, "land": "GEEN"}],
+                "aftrekposten": [{"bsn": "999993805", "persoonsgebonden_aftrek": 450000}],
+            },
+            "RVZ": {
+                "verzekeringen": [
+                    {"bsn": "999993805", "polis_status": "ACTIEF", "verdrag_status": "GEEN", "zorg_type": "BASIS"}
+                ]
+            },
+        },
+    },
```

Comment:
Dit profiel mist nog in de go backend. 
`machinev2/backend/cmd/serve_input.yaml`

Het is denk ik handiger om 1 profile.yaml te maken die beide engines gebruiken. 
Dan weten we zeker dat het in sync blijft.

maar voor een andere PR